modules/vba_documents/app/serializers/vba_documents/upload_serializer.rb

```diff
@@ -11,27 +11,25 @@ class UploadSerializer
     set_type :document_upload
     set_id :guid
 
-    attributes :guid, :code, :updated_at
+    attributes :guid, :status, :code
 
     attribute :detail do |object|
       detail = object.detail.to_s
       detail.length > MAX_DETAIL_DISPLAY_LENGTH ? "#{detail[0..MAX_DETAIL_DISPLAY_LENGTH - 1]}..." : detail
     end
 
-    attribute :uploaded_pdf do |object|
-      scrub_unnecessary_keys(object.uploaded_pdf) if object.uploaded_pdf
-    end
-
-    attribute :status do |object|
-      object.status == 'vbms' ? 'success' : object.status
-    end
```

Comment:
Fixing the order of fields for `uploaded_pdf`, and removing some old `status` code, which was left over from when `vbms` wasn't a status that was returned to consumers yet. Now, we simply return the contents of the `status` of the `UploadSubmission`, and the code wasn't hit because the V1 and V2 serializers overrode it.